src/components/content/ProtocolBadge.tsx

```diff
@@ -36,6 +36,11 @@ const kdeProtocolsUnstableBadgeTheme: BadgeTheme = {
     backgroundColor: 'bg-purple-100',
 }
 
+const cosmicProtocolsUnstableBadgeTheme: BadgeTheme = {
+    textColor: 'text-amber-100',
+    backgroundColor: 'bg-orange-400',
```

Comment:
I find the contrast between these two colors a bit hard-to-read:

![20240825_15h20m44s_grim](https://github.com/user-attachments/assets/5bf28c51-bfbc-4730-8522-43bb33a4f3c7)

I see what you mean about being difficult to find some colors that match the COSMIC logo though. I've tried a few combinations and the best options I could find are either `amber` or `yellow` (for both text and background).

### yellow

```yaml
textColor: 'text-yellow-100'
backgroundColor: 'bg-yellow-800'
```

![20240825_15h09m51s_grim](https://github.com/user-attachments/assets/a6f08006-0d25-4323-8f97-c435002f305a)

### amber

```yaml
textColor: 'text-amber-100'
backgroundColor: 'bg-amber-800'
```

![20240825_15h10m35s_grim](https://github.com/user-attachments/assets/9023de33-8e24-43fb-b3da-c1cb17546998)

Neither of them is great, but for these two options the contrast seems to be slightly better.  I don't have strong opinions on it though so I'm fine with either one of these 3 options in the end.